SPECS/python-paramiko/python-paramiko.spec

```diff
@@ -0,0 +1,736 @@
+%global srcname paramiko
+
+Name:          python-%{srcname}
+Version:       2.11.0
+Release:       2%{?dist}
+Summary:       SSH2 protocol library for python
+License:       LGPLv2+
+Distribution:  Mariner
+Vendor:        Microsoft Corporation
+URL:           https://github.com/paramiko/paramiko
+Source0:       %{url}/archive/refs/tags/%{version}.tar.gz#/%{srcname}-%{version}.tar.gz
+
+# Skip tests that would fail without SHA-1 signing support in backend (e.g. on EL-9)
```

Comment:
Please remove reference to EL-9 and re-evaluate if patch is necessary

Reply:
Done. Removed unwanted patches.